statsmodels/survey/summary_stats.py

```diff
@@ -0,0 +1,480 @@
+"""
+Methods for creating summary statistics and their SE for survey data.
+
+The main classes are:
+
+  * SurveyDesign : Parent class that creates attributes for easy 
+  implementation of other methods. Attributes include relabeled 
+  clusters, number of clusters per strata, etc.
+
+  * SurveyStat : implements methods to calculate the standard 
+  error of each statistic via either the bootstrap or jackknife
+
+  * SurveyMean : Calculates the mean of each column
+
+  * SurveyTotal : Calculates the total of each column
+
+  * SurveyQuantile: Calculates the specified quantile[s] of each column
+"""
+
+import numpy as np 
+import pandas as pd 
+
+class SurveyDesign(object):
+    """                                                                                                                                              
+    Creates attributes that allow easy implementation of survey methods.                                                                                                   
+                                                                                                                                                     
+    Parameters                                                                                                                                          
+    -------                                                                                                                                          
+    strata : array-like or None                                                                                                                                 
+        Strata for each observation. If none, an array 
```

Comment:
Probably should default to array of zeros.  Python is 0-based for most things.  It's just a label so should not matter.